srml/contracts/src/exec.rs

```diff
@@ -507,6 +534,12 @@ where
 
 		Ok(output)
 	}
+
+	/// Returns whether a contract, identified by address, is currently being executed.
```

Comment:
I believe more accurate would be to say that the contract identified by address is present on the call stack. Wdyt?

Reply:
I'll add that to the comment as well to clarify, but I don't think it's inaccurate to say that the contract is still being executed since it just made a call to another contract.

Reply:
Yep, sure, it wasn't inaccurate it just could be more accurate )